build.gradle

```diff
@@ -70,6 +70,8 @@ task zipReport(dependsOn: 'npmRunSpuild', type: Zip) {
 tasks.shadowJar.dependsOn('zipReport');
 tasks.run.dependsOn('zipReport');
```

Comment:
Can check if this is no longer necessary?

Reply:
Still required.

`dependsOn` - sets task dependencies. Executing `run` would require that `zipReport` be executed first.

`shouldRunAfter` - sets task ordering. Executing `run` will be execute after `zipReport`.

Reply:
>`dependsOn` - sets task dependencies. Executing `run` would require that `zipReport` be executed first.

Hmmm, in this case why previously using `dependsOn` does not work, since `zipReport` must have been executed first before `run`? 🤔

Reply:
According to [Gradle](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:ordering_tasks), 

```
Note that “B.mustRunAfter(A)” or “B.shouldRunAfter(A)” does not imply any execution
dependency between the tasks:

It is possible to execute tasks A and B independently. The ordering rule only has an
effect when both tasks are scheduled for execution.
```
🤔